### PR TITLE
Update KIDs

### PR DESCRIPTION
**Pull request by reza-fs**

### CODE REVIEW

File: src/templates/static/legal/regulation.jsx [hunk 1]
```diff
@@ -154,11 +154,9 @@ const Regulation = () => {
...
-                            <FillBox id='touch-no-touch_fillbox' padding='4' center border='border-dark-gray' href={it.url_for(`/download/key_information_document/${lang}/Touch_No_Touch.pdf`)} target='_blank' text={it.L('Touch/No Touch')} />
-                            <FillBox id='in-and-out_fillbox' padding='4' center border='border-dark-gray' href={it.url_for(`/download/key_information_document/${lang}/In_and_Out.pdf`)}     target='_blank' text={it.L('In/Out')} />
-                            <FillBox id='cfd_fillbox' padding='4' center className='margin-right-0' border='border-dark-gray' href={it.url_for(`/download/key_information_document/${lang}/CFD.pdf`)} target='_blank' text={it.L('CFD')} />
-                            <FillBox id='fx_fillbox' padding='4' center className='margin-left-0'  border='border-dark-gray' href={it.url_for(`/download/key_information_document/${lang}/FX.pdf`)}  target='_blank' text={it.L('FX')} />
+                            <FillBox id='crypto_fillbox' padding='4' center border='border-dark-gray' href={it.url_for(`/download/key_information_document/${lang}/Crypto.pdf`)}        target='_blank' text={it.L('Crypto')} />
```
Comment (negarn) [inline src/templates/static/legal/regulation.jsx]: `target` has too much empty space behind it

File: src/templates/static/legal/regulation.jsx [hunk 2]
```diff
@@ -154,11 +154,9 @@ const Regulation = () => {
...
+                            <FillBox id='cfd_fillbox' padding='4' center className='margin-right-0' border='border-dark-gray' href={it.url_for(`/download/key_information_document/${lang}/Commodities.pdf`)} target='_blank' text={it.L('CFD')} />
+                            <FillBox id='fx_fillbox' padding='4' center className='margin-left-0'  border='border-dark-gray' href={it.url_for(`/download/key_information_document/${lang}/Forex.pdf`)}  target='_blank' text={it.L('FX')} />
```
Comment (negarn) [inline src/templates/static/legal/regulation.jsx]: same here

File: src/templates/static/legal/metatrader/terms_and_conditions.jsx [hunk 3]
```diff
@@ -133,6 +133,8 @@ const TermsAndConditions = () => (
                 <ol>
                     <li>{it.L('If the client wishes to withdraw funds from their MT5 Real Account and close their MT5 Real Account, they may notify the Company by contacting [_1] helpdesk [_2]. The client\'s MT5 Real Account may be closed if they do not have any Open Positions and all the amounts that were due to the Company have been settled.', it.website_name, '<a href=\'mailto:support@binary.com\'>support@binary.com</a>')}</li>
                     <li>{it.L('If the client\'s [_1] account is closed, their MT5 Real Account will be automatically closed as well.', it.website_name)}</li>
+                    <li>{it.L('Client\'s demo accounts will be deleted after 365 days of inactivity. [_1] Open positions and pending orders for demo accounts will be deleted after 180 days of inactivity.', '<br />')}</li>
```
Comment (negarn) [inline src/templates/static/legal/metatrader/terms_and_conditions.jsx]: why not add the br and then a new translation tag for the second sentence?

Comment (reza-fs) [reply src/templates/static/legal/metatrader/terms_and_conditions.jsx]: Oh, you are right. I got lost in syntax.

File: src/templates/static/legal/metatrader/terms_and_conditions.jsx [hunk 4]
```diff
@@ -133,6 +133,9 @@ const TermsAndConditions = () => (
                 <ol>
                     <li>{it.L('If the client wishes to withdraw funds from their MT5 Real Account and close their MT5 Real Account, they may notify the Company by contacting [_1] helpdesk [_2]. The client\'s MT5 Real Account may be closed if they do not have any Open Positions and all the amounts that were due to the Company have been settled.', it.website_name, '<a href=\'mailto:support@binary.com\'>support@binary.com</a>')}</li>
                     <li>{it.L('If the client\'s [_1] account is closed, their MT5 Real Account will be automatically closed as well.', it.website_name)}</li>
+                    <li>{it.L('Client\'s demo accounts will be deleted after 365 days of inactivity.')}</li>
+                    <br />{it.L('Open positions and pending orders for demo accounts will be deleted after 180 days of inactivity.')}
```
Comment (negarn) [inline src/templates/static/legal/metatrader/terms_and_conditions.jsx]: will it be inside the `</li>`?

Comment (reza-fs) [reply src/templates/static/legal/metatrader/terms_and_conditions.jsx]: No, it is not another item, it's just a line break.

**Pull request merged**